packages/flutter_tools/lib/src/commands/apk.dart

```diff
@@ -296,6 +297,9 @@ int _buildApk(
     ensureDirectoryExists(finalApk.path);
     builder.align(unalignedApk, finalApk);
 
+    File apkShaFile = new File('$outputFile.sha');
```

Comment:
Where do we store this file?  We should put it in the same directory as the APK (i.e., in the `build` directory).  Also, industry practice is to use the file extension `sha1` for this purpose.  The `sha` extension has other uses on Windows.